src/main/kotlin/racing/car/Car.kt

```diff
@@ -0,0 +1,29 @@
+package racing.car
+
+import racing.utils.RandomNumberGenerator
+
+data class Car(
+    private val _carName: String,
+    private var _moveCount: Int = 0,
+) {
+
+    val carName: String
+        get() { return _carName }
+
+    val moveCount: Int
+        get() { return _moveCount }
+
+    fun move() {
+        if (canMoveForward(RandomNumberGenerator.getRandomNumber())) {
+            moveForward()
+        }
+    }
+
+    fun canMoveForward(number: Int) = number >= MOVE_CONDITION
+
+    fun moveForward() = this._moveCount++
+
+    companion object {
+        const val MOVE_CONDITION = 4
```

Comment:
```suggestion
        private const val MOVE_CONDITION = 4
```
외부에 노출되지 않아도 될 값으로 보입니다.